core/src/main/java/org/elasticsearch/common/ParseField.java

```diff
@@ -29,6 +29,12 @@
  * Holds a field that can be found in a request while parsing and its different variants, which may be deprecated.
  */
 public class ParseField {
+    // todo reuse these commonly used ParseField objects instead of duplicating them in other classes
+    public static final ParseField FIELD_FIELD = new ParseField("field");
+    public static final ParseField FORMAT_FIELD = new ParseField("format");
+    public static final ParseField MISSING_FIELD = new ParseField("missing");
+    public static final ParseField TIME_ZONE_FIELD = new ParseField("time_zone");
+    public static final ParseField VALUE_TYPE_FIELD = new ParseField("value_type", "valueType");
```

Comment:
+1 but I would check with @clintongormley whether we are happy making this breaking change in 5.0. However, we haven't actually formally removed camelCase in ParseField IIRC so adding this extra name here might beunnecessary anyway